### PR TITLE
Get asset data

### PR DESCRIPTION
**Pull request by aronhoyer**
Adds backend support for getting asset data

### CODE REVIEW

**Review by ovineq — CHANGES_REQUESTED** [review]
I think it's time to define types for the returned objects from these API calls. The `any` type isn't very helpful, as it bypasses the typescript compiler. 
Having defined types will help newcomers explore the available data, and should also help in reviewing PRs. 😄

File: pages/api/assets/[id]/overview.ts [hunk 1]
```diff
@@ -0,0 +1,33 @@
...
+        headers: { authorization: req.headers.authorization },
+        query: { assetId: req.query.id },
+      })
+
+      const attrs = attrsRes.body.results.filter((attr: any) => [
+        "purpose",
+        "timeliness",
+        "description",
+      ].includes(attr.type.name.toLowerCase())).map((attr: any) => ({
+        type: attr.type.name.toUpperCase().split(" ").join("_"),
```
[the remark this answers is not in the document]
Comment (aronhoyer) [reply pages/api/assets/[id]/overview.ts:22]: it absolutely would 🥲

**Comment by aronhoyer** [comment]
> I think it's time to define types for the returned objects from these API calls

@ovineq agree! should types be global or do we want to manually import them for each use?

**Comment by ovineq** [comment]
> > I think it's time to define types for the returned objects from these API calls
> 
> @ovineq agree! should types be global or do we want to manually import them for each use?

I'm most familiar with them being local and imported. I also have a preference for namespaces for external type definitions. E.g. if I see a type `Collibra.Asset` in our code base, I know that it is most likely a data transfer object from the Collibra API.

We might at some point define our own internal `Asset` that might not have the same properties as the external definition. Having the root namespace for our own definitions is a boon.

File: config.ts [hunk 2]
```diff
@@ -1,5 +1,6 @@
 export const config = Object.freeze({
   BASE_URL: process.env.BASE_URL ?? "http://localhost:3000",
+  COLLIBRA_BASE_URL: process.env.NEXT_PUBLIC_COLLIBRA_BASE_URL ?? "",
 
   /* Auth config */
   AUTH_AUTHORITY: process.env.NEXT_PUBLIC_AUTH_AUTHORITY ?? "",
```
[the remark this answers is not in the document]
Comment (ovineq) [reply config.ts:6]: Talked in the standup this morning about fixing this at a later date.

[the remark this answers is not in the document]
Comment (ovineq) [reply config.ts:6]: Talked in the standup this morning about fixing this at a later date.

**Pull request merged**